clusters/build-clusters/01_cluster/machine_config/worker-m5d4x_machineconfigpool.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: machineconfiguration.openshift.io/v1
+kind: MachineConfigPool
+metadata:
+  name: worker-m5d4x
+spec:
+  machineConfigSelector:
+    matchExpressions:
+      - {key: machineconfiguration.openshift.io/role, operator: In, values: [worker, m5d4x]}
+  nodeSelector:
+    matchLabels:
+      beta.kubernetes.io/instance-type: "m5d.4xlarge"
```

Comment:
This might work, I'm not sure.  I think what I did was
node-role.kubernetes.io/worker-m5d4x: ""

@cgwalters can probably better explain the linkages here.

Reply:
Done.